tests/e2e/cli/action/test_action_classification.py

```diff
@@ -96,6 +96,8 @@ def test_otx_export(self, template, tmp_dir_path):
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_otx_eval_openvino(self, template, tmp_dir_path):
         tmp_dir_path = tmp_dir_path / "action_cls"
+        if template.model_template_id == "Custom_Action_Classification_MoViNet":
+            pytest.xfail("CVS-112824: MoViNet inference fails in OV 2023.0")
```

Comment:
Could you made a GH issue and please use the GH number instead of CVS number since OTX is open-source

Reply:
Updated, we already have an issue on github